log/field.go

```diff
@@ -0,0 +1,204 @@
+package log
```

Comment:
Funny... I've been thinking of semver bumps in terms of clients of the API (rather than implementations), but I see what you mean. We'd have to resign the symbolic importance of major version numbers :-/